client/data/users/use-users.js

```diff
@@ -0,0 +1,52 @@
+/**
+ * External dependencies
+ */
+import { useInfiniteQuery } from 'react-query';
+import { uniqueBy } from '@automattic/js-utils';
+
+/**
+ * Internal dependencies
+ */
+import wpcom from 'calypso/lib/wp';
+
+export const DEFAULT_NUMBER = 100;
```

Comment:
Nit: naming here looks odd; could make it a bit more clear and specific? `DEFAULT_PER_PAGE` or something along these lines perhaps?

Reply:
This constant is gone as of https://github.com/Automattic/wp-calypso/commit/0509f4ca718d8cee1f0d5fd3a4814ffd1077ae00